openwisp_radius/base/models.py

```diff
@@ -860,7 +860,7 @@ def clean(self):
             )
         if self.strategy == 'csv':
             validate_csvfile(self.csvfile.file)
-        super(AbstractRadiusBatch, self).clean()
+        super().clean()
```

Comment:
This is a random chore, cleanup, not related to the issue but just saw it!

Reply:
thanks :+1: